source/jsonnet/england-wales/ccs/lib/rules.libsonnet

```diff
@@ -49,6 +49,19 @@ local livingAtCurrentAddress = {
   value: 'Yes, living at this address',
 };
 
+local isHouseholdAddress = {
+  id: 'you-live-here-answer',
+  condition: 'equals',
+  value: 'Yes',
+};
+
+
+local isNotHouseholdAddress = {
+  id: 'you-live-here-answer',
+  condition: 'equals',
+  value: 'No',
+};
+
```

Comment:
It would be clearer if the boolean was named closer to the question/answer, something like `livesAtHouseholdAddress` and `doesntLiveAtHouseholdAddress`

Reply:
Changed to these suggested names, thanks👍